modules/gdscript/language_server/gdscript_workspace.cpp

```diff
@@ -231,18 +231,13 @@ Error GDScriptWorkspace::initialize() {
 			class_symbol.children.push_back(symbol);
 		}
 
-		Vector<DocData::PropertyDoc> properties;
-		properties.append_array(class_data.properties);
-		const int theme_prop_start_idx = properties.size();
-		properties.append_array(class_data.theme_properties);
-
 		for (int i = 0; i < class_data.properties.size(); i++) {
```

Comment:
This code had a type before this PR. While both properties and theme properties were collected into `properties` local vector, the iterator was using `class_data.properties` instead. So theme items were never handled in any way.

Since properties and theme properties no longer share a type I've split them, but as this wasn't working properly before I'm not sure if there aren't any side effects. So, beware?

Reply:
I'm not familiar with the LSP but the rationale makes sense to me. (FYI @Razoric480)